tools/rtrclient.c

```diff
@@ -140,19 +212,144 @@ static bool is_resolveable_host(const char *str)
 	return true;
 }
 
-#ifdef RTRLIB_HAVE_LIBSSH
+static bool is_file(const char *str)
+{
+	struct stat stat_buf;
+
+	if (stat(str, &stat_buf) == -1 )
```

Comment:
remove space after `-1`

Reply:
travis (check patch in there) found that too and one or two more

Reply:
already done